### PR TITLE
Add translated user documentation

### PR DESCRIPTION
**Pull request by nrc**
Translated from https://github.com/pingcap/docs-cn/pull/2878

I haven't made the prose read very well, this is mostly machine-translated, then I've fixed up the formatting, etc. I'll file a follow-up issue to improve the text.

PTAL @lucklove

### CODE REVIEW

File: doc/user/package.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Component packaging
+
+When you want to add a new component, or add a version of an existing component, you need to use tar to package the relevant file and then pass it to the mirror repository, using tar to package is not a difficult thing, the trouble is that you need to update the repository's meta-information, to avoid updating meta-information when corrupting the information of existing components. So the package component takes on this task.
+
+```bash
+[root@localhost ~]# tiup package --help
```
Comment (lucklove) [inline doc/user/package.md:6]: The English version is:
```
Package a tiup component and generate package directory

Usage:
  tiup package target [flags]

Flags:
  -C, -- string          Change directory before compress
      --arch string      Target ARCH of the package (default "amd64")
      --desc string      Description of the package
      --entry string     Entry point of the package
  -h, --help             help for tiup
      --hide tiup list   Don't show the component in tiup list
      --name string      Name of the package
      --os string        Target OS of the package (default "darwin")
      --release string   Version of the package
      --standalone       Can the component run standalone
```

File: doc/user/playground.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- Since no version is specified, TiUP looks for the latest version of the installed playground first, assuming the latest version is v0.0.6, which is equivalent to tiup playground:v0.0.6.
+- If the playground component has never had any version installed, TiUP will install the latest stable version before starting the runtime instance.
+- Since playground does not specify the version of each TiDB/PD/TiKV component, by default it will use the latest release version of each component, and assuming the current version is v4.0.0-rc, this command is equivalent to tiup playground:v0.0.6 v4.0.0-rc
+- Since playground also does not specify the number of components, by default it starts a minimized cluster of 1 TiDB, 1 TiKV and 1 PD
+- After starting each component in turn, the playground will tell you that it started successfully and tell you some useful information, such as how to connect the cluster through the MySQL client, how to access the dashboard
+
+The command line arguments for playground state:
+
```
Comment (lucklove) [inline doc/user/playground.md:21]: The English version is:
```
Usage:
  tiup playground [version] [flags]

Flags:
      --db int                   TiDB instance number (default 1)
      --db.binpath string        TiDB instance binary path
      --db.config string         TiDB instance configuration file
  -h, --help                     help for tiup
      --host string              Playground cluster host (default "127.0.0.1")
      --kv int                   TiKV instance number (default 1)
      --kv.binpath string        TiKV instance binary path
      --kv.config string         TiKV instance configuration file
      --monitor                  Start prometheus component
      --pd int                   PD instance number (default 1)
      --pd.binpath string        PD instance binary path
      --pd.config string         PD instance configuration file
      --tiflash int              TiFlash instance number
      --tiflash.binpath string   TiFlash instance binary path
      --tiflash.config string    TiFlash instance configuration file
```

**Comment by nrc** [comment]
@lucklove thanks for the review and the help message texts. I've updated the PR, PTAL.

**Comment by yikeke** [comment]
The 7 English TiUP documents are now online on the PingCAP website. Check it out: https://pingcap.com/docs/dev/reference/tools/tiup/overview/ 

FYI @nrc

**Pull request merged**